src/ShortcutGuide.tsx

```diff
@@ -22,282 +27,63 @@ const ShortcutGuide: React.FC = () => {
-										<th className="a11y_explanation">Move backward in a page element</th>
-									</tr>
-									<tr
-										className="a11y_row"
-										aria-label="Move between series of data inside the chart: Alt (option) + M"
-										role={"listitem"}
-									>
-										<th className="a11y_shortcut">Alt (option) + M</th>
-										<th className="a11y_explanation">
-											Move between series of data inside the chart
-										</th>
-									</tr>
-									<div className="a11y_empty_row"></div>
-									<tr className="a11y_row">
-										<th></th>
-										<th
-											className="a11y_title"
-											aria-label="Section: Chart navigation shortcuts"
-											role={"listitem"}
-										>
-											Chart navigation shortcuts
-										</th>
-									</tr>
-									<tr
-										className="a11y_row"
-										aria-label="Jump to the beginning of a chart: Home or Alt (option) + Q "
-										role={"listitem"}
-									>
-										<th className="a11y_shortcut">Home or Alt (option) + Q</th>
-										<th className="a11y_explanation">Jump to the beginning of a chart</th>
-									</tr>
-									<tr
-										className="a11y_row"
-										aria-label="Jump to the end of a chart: End or Alt (option) + W "
-										role={"listitem"}
-									>
-										<th className="a11y_shortcut">End or Alt (option) + W</th>
-										<th className="a11y_explanation">Jump to the end of a chart</th>
-									</tr>
-									<tr
-										className="a11y_row"
-										aria-label="Define the number of data points to be jumped at a time: Alt (option) + X"
-										role={"listitem"}
-									>
-										<th className="a11y_shortcut">Alt (option) + X</th>
-										<th className="a11y_explanation">
-											Define the number of data points to be jumped at a time
-										</th>
-									</tr>
-									<tr
-										className="a11y_row"
-										aria-label="Add one number to the data points to be jumped at a time: Plus symbol"
-										role={"listitem"}
-									>
-										<th className="a11y_shortcut">+</th>
-										<th className="a11y_explanation">
-											Add one number to the data points to be jumped at a time
-										</th>
-									</tr>
-									<tr
-										className="a11y_row"
-										aria-label="Subtract one number to the data points to be jumped at a
-                      time: Minus symbol"
-										role={"listitem"}
-									>
-										<th className="a11y_shortcut">-</th>
-										<th className="a11y_explanation">
-											Subtract one number to the data points to be jumped at a time
-										</th>
-									</tr>
-									<div className="a11y_empty_row"></div>
-								</tbody>
-							</table>
+					<hr className="shortcut-guide__break" />
+					{[guideData].map((sectionGroup, groupIndex) => (
+						<div className="shortcut-guide__body" key={groupIndex}>
+							{sectionGroup.map((section) => (
+								<>
+									<div className="shortcut-guide__section">
```

Comment:
We may be overcomplicating the semantics and using too many nested array maps. The current approach uses override techniques to add new roles to generic HTML elements like divs.
Key questions to consider:

1. If we're adding the `listitem` role, should we use a native list instead?
2. If we use a native list, which type is most appropriate: unordered, ordered, or description list?

I propose using a description list (`<dl>`), as its purpose aligns well with presenting a set of metadata in a concise manner.
Suggested implementation:

```suggestion
    <div className="shortcut-guide__body">
        <div className="shortcut-guide__section">
            <h3 id="something-unique-for-your-list" className="shortcut-guide__cell shortcut-guide__cell--title">{section.title}</h3>
            <dl class="shortcut-guide__list" aria-labelledby="something-unique-for-your-list">
            <div className="shortcut-guide__row">
                <dt className="shortcut-guide__cell shortcut-guide__cell--shortcut">{shortcut.keys}</dt>
                <dd className="shortcut-guide__cell shortcut-guide__cell--explanation">{shortcut.description}</dd>
            </div>
            </dl>
        </div>
    </div>
```
Benefits of this approach:
- No overridden semantics or short-circuits
- Cleaner, more readable HTML
- Improved screen reader navigation (users can jump between headings)
- Eliminates the need for forced tab index

To address the nested maps issue:
- Break the component into smaller, more manageable parts
- This will help prevent unnecessary re-computations on re-renders
- Remember to include a unique key prop for each top-level HTML element inside a map in React (currently only `groupIndex` is used)

Reply:
It also looks like the way we are controlling visibility is an issue as well. For this use case we should implement a dialog (https://web.dev/learn/html/dialog).